src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/ModelMetadata.cs

```diff
@@ -2,7 +2,9 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Collections;
 using System.Collections.Generic;
+using System.Reflection;
```

Comment:
I think this is fine for a first cut, but we might want to clean it up further by moving those things to core one way or another.